.pre-commit-config.yaml

```diff
@@ -42,6 +42,12 @@ repos:
   - id: absolufy-imports
     exclude: docs/plasmapy_sphinx
 
+- repo: https://github.com/mattlqx/pre-commit-search-and-replace
+  rev: v1.0.5
+  hooks:
+  - id: search-and-replace
+    exclude: changelog/.*|docs/changelog/.*
```

Comment:
This uses regular expressions, where the `.*` means any character, any number of times; and the `|` means "or".